plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerDelegate.java

```diff
@@ -65,12 +68,27 @@ public IStatus canModifyModules(IModule[] add, IModule[] remove) {
           if (status != null && !status.isOK()) {
             return status;
           }
+          status = hasAppEngineStandardFacet(module);
```

Comment:
Reusing local variables is error prone in the face of future changes. I suggest using two different IStatus variables.